src/utils/isScrolledTo/isScrolledTo.ts

```diff
@@ -0,0 +1,72 @@
+enum ScrollEdge {
+  top,
+  left
+}
+
+enum Dimension {
+  x,
+  y
+}
+
+// eslint-disable-next-line consistent-return
+const isScrolledToCloserEdge = (scrollEdge: ScrollEdge) => ($container: HTMLElement) => {
```

Comment:
API of this function is not consistent with `isScrolledToFartgerEdge`. This function takes `scrollEdge` as param an the other takes `dimension` and also uses `$scroll`. Care to explain?